README.md

```diff
@@ -48,17 +47,6 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235654740-ecbf1218-8a38-419b-b04f-ffb922b54810.png)

**[OPTIONAL]**

- [ ] It would be nice if you put some exciting features in your repo that you are gonna implement in your project in the near future to excite the viewers (especially recruiters) of your project and build your reputation in the job market 👍 You can add something like ⬇️

* Reduce Space and Time Complexity
* Normalize the database upto some other levels

Reply:
Thank you @HAMMAS-SALEEM